src/imageTools/probe.js

```diff
@@ -68,6 +68,18 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
         return suv;
     }
     ///////// BEGIN IMAGE RENDERING ///////
+    function pointNearTool(data, coords) {
+        var point = {
+            left: data.handles.end.x,
+            top: data.handles.end.y,
+            width: 0,
+            height: 0
+        };
+
+        var distanceToPoint = cornerstoneMath.rect.distanceToPoint(point, coords);
```

Comment:
Erik, is the last change OK? Do I need to do anything else to accept my PR?

Not in a hurry, just asking :)


On 19-May-2015 4:35 PM, Erik Ziegler wrote:
>
> In src/imageTools/probe.js 
> <https://github.com/chafey/cornerstoneTools/pull/41#discussion_r30598762>:
>
> > @@ -68,6 +68,18 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
> >          return suv;
> >      }
> >      ///////// BEGIN IMAGE RENDERING ///////
> > +    function pointNearTool(data, coords) {
> > +        var point = {
> > +            left: data.handles.end.x,
> > +            top: data.handles.end.y,
> > +            width: 0,
> > +            height: 0
> > +        };
> > +
> > +        var distanceToPoint = cornerstoneMath.rect.distanceToPoint(point, coords);
>
> Great! More collaborators is always good. @chafey 
> <https://github.com/chafey> deserves all the credit
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/chafey/cornerstoneTools/pull/41/files#r30598762>.
>